se/__init__.py

```diff
@@ -0,0 +1,9 @@
+from commands import *
```

Comment:
is this necessary?  i think this is merging all the modules into one namespace.  however, when accessing you are doing se.module.X so that makes me think this isn't necessary (but you probably still need to have an empty __init__.py file)